### DISCUSSION TITLE
Plugin folder location

### DISCUSSION THREAD

**Discussion by bzuidgeest** [quartznet/quartznet#1149] [Q&A] [answered]
Sorry to post this as a bug report, but I just started working with quartz.net and have a seemingly simple question I cannot find the answer to.

The quartz plugins seem to be loaded from a plugin folder of the main application folder. But this is not usable in my case as the application already uses that folder.

Is there a way to tell quartz where to look for plugin dlls?

**Comment by lahma** [comment, marked as answer]
> Sorry to post this as a bug report, but I just started working with quartz.net and have a seemingly simple question I cannot find the answer to.

So I went ahead and converted this to discussion.

> The quartz plugins seem to be loaded from a plugin folder of the main application folder. But this is not usable in my case as the application already uses that folder.
> 
> Is there a way to tell quartz where to look for plugin dlls?

I'm not aware of Plugin folder requirement. You configure plug-ins by providing their type on configuration and .NET type loading handles the rest, looking in the application folder by default. You could configure type loader to look from additional paths but that's not the default.

**Reply by bzuidgeest** [reply]
> So I went ahead and converted this to discussion.

Thanks, still to used to posting everything in issues I guess

>  .NET type loading handles the rest

As my application has it's own plugin handling and the scheduler is going to be in a plugin itself, I might have gotten some things crossing over on each other then. I was hoping quartz scheduler would have a way to set the folder where it looks for its own plugins directly. So they could be in a subfolder of its own pluginfolder..... :).
